app/server/routes/api.js

```diff
@@ -181,34 +171,11 @@ module.exports = function(router) {
    * POST - Decline an acceptance.
    */
   router.post('/users/:id/decline', isOwnerOrAdmin, function(req, res){
-    var confirmation = req.body.confirmation;
-    var id = req.params.id;
+    const id = req.params.id;
 
     UserController.declineById(id, defaultResponse(req, res));
   });
 
-  /**
```

Comment:
So we're not letting them update/reset their password?

Reply:
We are, but we are only doing it through email

Reply:
Gotcha!